pkg/ddevapp/drupal.go

```diff
@@ -249,13 +250,9 @@ if (getenv('IS_DDEV_PROJECT') == 'true') {
 	return nil
 }
 
-// getDrupalUploadDir will return a custom upload dir if defined, returning a default path if not.
-func getDrupalUploadDir(app *DdevApp) string {
-	if app.UploadDir == "" {
-		return "sites/default/files"
-	}
-
-	return app.UploadDir
+// getDrupalUploadDirs will return the default paths.
+func getDrupalUploadDirs(_ *DdevApp) UploadDirs {
+	return UploadDirs{"sites/default/files"}
```

Comment:
Consider changing to UploadDirs{"sites/default/files", "../private"}

Reply:
Done